server.go

```diff
@@ -249,14 +252,58 @@ func isBatch(message json.RawMessage) bool {
 	return false
 }
 
+// ConvertToObject convert json array into object using key by index from keys array
+func ConvertToObject(keys []string, params json.RawMessage) (json.RawMessage, error) {
+	paramCount := len(keys)
+
+	rawParams := []json.RawMessage{}
+	if err := json.Unmarshal(params, &rawParams); err != nil {
+		return nil, err
+	}
+
+	if paramCount != len(rawParams) {
```

Comment:
params could be optional, maybe we will ignore it right now, but we need comment about it